packages/plugins/Zoom/src/components/Zoom.vue

```diff
@@ -40,26 +40,40 @@
       </template>
       <span>{{ $t('common:plugins.zoom.out') }}</span>
     </v-tooltip>
+    <v-tooltip left :disabled="hasSmallDisplay">
+      <template #activator="{ on, attrs }">
+        <div v-bind="attrs" v-on="on">
+          <ZoomSlider v-if="addZoomSlider"></ZoomSlider>
```

Comment:
```suggestion
    <v-tooltip v-if="addZoomSlider" left :disabled="hasSmallDisplay">
      <template #activator="{ on, attrs }">
        <div v-bind="attrs" v-on="on">
          <ZoomSlider></ZoomSlider>
```

Reply:
The `v-if` is still present on the `ZoomSlider` and not on `v-tooltip`

Reply:
has been changed